modules/local/svprep/call/main.nf

```diff
@@ -0,0 +1,108 @@
+process GRIDSS_CALL {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmftools-sv-prep:1.2.4--hdfd78af_0' :
+        'biocontainers/hmftools-sv-prep:1.2.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bams), path(bams_filtered), path(assemble_dir), val(labels)
+    path gridss_config
+    path genome_fasta
+    path genome_fai
+    path genome_dict
+    path genome_gridss_index
+    path blocklist
+
+    output:
+    tuple val(meta), path('gridss_call/sv.svprep.gridss.vcf.gz'), emit: vcf
+    path 'versions.yml'                                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    def config_arg = gridss_config ? "--configuration ${gridss_config}" : ''
+    def output_dirname = 'gridss_call'
+    def labels_list = labels instanceof List ? labels : [labels]
+    def labels_arg = labels_list.join(',')
+    // NOTE(SW): Nextflow implicitly casts List<TaskPath> to an atomic TaskPath, hence the required check below
+    def bams_list = bams instanceof List ? bams : [bams]
+    def bams_arg = "--bams ${bams_list.join(',')}"
+    def bams_filtered_list = bams_filtered instanceof List ? bams_filtered : [bams_filtered]
+    def bams_filtered_arg = "--filtered_bams ${bams_filtered_list.join(',')}"
+
+    // JVM heap for other tasks must be no greater than 1/4 of task memory, defaults to 1 GB if not provided
+    def otherJvmHeap = Math.min(
+        Math.round(task.memory.bytes * 0.25),
+        task.ext.otherJvmHeap ? task.ext.otherJvmHeap.bytes : 1.GB.bytes
+    )
+
+    """
+    # Create shadow directory with file symlinks of GRIDSS 'working' dir to prevent cache invalidation
+    # NOTE: for reasons that elude me, NF doesn't always stage in the workingdir; remove if it is present
```

Comment:
```suggestion
    # NOTE(SW): for reasons that elude me, NF doesn't always stage in the workingdir; remove if it is present
```

Reply:
- Change made in https://github.com/nf-core/oncoanalyser/pull/67